root/layout/menu.tt

```diff
@@ -283,27 +283,6 @@
         </ul>
     </li>
     <li class="contact">
-        <a href="[% doc_link('Contact_Us') %]">[% l('Contact Us') %]</a>
-        <ul>
-            <li>
-                <a href="[% doc_link('Communication/Mailing_Lists') %]">[% l('Mailing Lists') %]</a>
```

Comment:
Perhaps best to leave it until we decide what to do with those pages. Making them disappear without a trace would be confusing, since people have used the lists for years, and we could e.g. link to an archived version from there or something.